test/controllers/users/sessions_controller_test.rb

```diff
@@ -231,4 +231,65 @@ class Users::SessionsControllerTest < ActionDispatch::IntegrationTest
       assert_redirected_to root_url(subdomain: @restarone_subdomain.name)
     end
   end
+
+  test 'show allow #login without otp if enable_2fa is set to false' do
+    Subdomain.current.update(enable_2fa: false)
+    @user.update(global_admin: true, current_sign_in_ip: '172.18.1.0', last_sign_in_ip: '172.18.1.0')
+    payload = {
+      user: {
+        email: @user.email,
+        password: '123456'
+      }
+    }
+    post user_session_url, params: payload
+    assert_redirected_to admin_subdomain_requests_url
+    # assert_redirected_to comfy_admin_cms_site_pages_path(site_id: Comfy::Cms::Site.first.id)
+    follow_redirect!
+    assert_template :index
+  end
+
+test 'show deny #login with wrong otp if enable_2fa is set to true' do
```

Comment:
@ajaystha42 what is this testing for? `show deny #login` doesn't make sense to me